src/main/java/org/example/competitionService/CompetitionService.java

```diff
@@ -0,0 +1,44 @@
+package org.example.competitionService;
```

Comment:
why did you created package competitionService ? Just to put one class CompetitionService ?